pybamm/step/_steps_util.py

```diff
@@ -37,7 +37,7 @@ class _Step:
         or "resistance".
     value : float
         The value of the step, corresponding to the type of step. Can be a number, a
-        2-tuple (for cccv_ode), or a 2-column array (for drive cycles)
+        2-tuple (for cccv_ode), or a 2-column array. Can pass list as argument (for drive cycles)
```

Comment:
Just went to the original issue, the problem is that this comment that the value can be a 2-column array is not shown in [the docs](https://docs.pybamm.org/en/latest/source/api/experiment/experiment_steps.html) because it is not written in `steps.py`. So I would suggest reverting this change and adding the extra info in the docstrings of `steps.py`.